anti_patterns/deny-warnings.md

```diff
@@ -0,0 +1,101 @@
+# `#![deny(warnings)]`
+
+## Description
+
+A well-intentioned crate author wants to ensure their code builds without
+warnings. So they annotate their crate root with
```

Comment:
Reading the rendered view, it's not obvious that the next example finishes this sentence. I'd rather have this written out in full instead.